src/mumble/AudioInput.cpp

```diff
@@ -149,8 +149,9 @@ AudioInput::~AudioInput() {
 	wait();
 
 #ifdef USE_OPUS
-	if (opusState)
-		opus_encoder_destroy(opusState);
+	if (opusState) {
```

Comment:
Oh, here we actually did have an opusState check. That doesn't make much sense if we simply use opusState above as if it were always usable...

We should probably add the error check and set null in a commit before this one

Reply:
I think it would be ideal to do it in a dedicated pull request, as this one should just make the shared library work.

Reply:
On one access of oCodec we check oCodec, on another we check opusState. I don’t like that asymmetry. Definitely not "ideal". But w/e

Reply:
I don't like it either, but we will fix it in a dedicated pull request.